core/src/avm2/globals/XML.as

```diff
@@ -44,6 +45,19 @@ package {
         AS3 native function text():XMLList;
         AS3 native function toString():String;
 
+        AS3 function hasComplexContent():Boolean {
+            return !this.hasSimpleContent();
```

Comment:
This is not correct. hasComplexContent isn't simply the inverse of hasSimpleContent. (i.e. comments are neither complex nor simple).

See [ECMA-357](https://www.ecma-international.org/publications-and-standards/standards/ecma-357/)